compose/logging/README.MD

```diff
@@ -2,55 +2,59 @@
 
 ## Summary
 
-- We are using Elastic Search and Kibana for logging indexing and visualization.
-- Probably the best source of truth here should be the `docker-compose.yaml` file 
+- We are using the [Elastic stack](https://www.elastic.co/webinars/introduction-elk-stack) for logging, indexing and visualization.
+- Probably the best source of truth here should be the docker-compose file `logging.yaml` 
 
 ## Prerequisites
 
-- You should be familiar with what Elastic Search and Kibana are and how they operate.
-
-## Installation
-
-None
+- You should be familiar with what [Elastic stack](https://www.elastic.co/webinars/introduction-elk-stack) is and how it operates.
 
 ## Running and testing
 
 ### Running
 
-```bash
+```docker
```

Comment:
nit. I'm pretty sure that should be **bash** since you are running a bash command  of `docker` and not setting up a markdown with the docker language (ex. for a docker file)